src/VisualStudio/Core/Def/Implementation/ExtractInterface/ExtractInterfaceDialog.xaml

```diff
@@ -59,17 +62,51 @@
                      Background="{DynamicResource {x:Static SystemColors.ControlBrush}}"
                      Padding="{StaticResource ResourceKey=textboxPadding}"
                      GotFocus="SelectAllInTextBox"/>
-            <Label x:Uid="FileNameLabel" 
-                   x:Name="FileNameLabel"
-                   Content="{Binding ElementName=dialog, Path=NewFileName}" 
-                   Padding="{StaticResource ResourceKey=labelPadding}"
-                   Target="{Binding ElementName=fileNameTextBox}"/>
-            <TextBox x:Uid="FileNameTextBox" 
-                     AutomationProperties.LabeledBy="{Binding ElementName=FileNameLabel}"
-                     Text="{Binding FileName, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" 
-                     Padding="{StaticResource ResourceKey=textboxPadding}"
-                     Name="fileNameTextBox"
-                     GotFocus="SelectAllInTextBox"/>
+
+            <GroupBox x:Uid="DestinationSelectionGroupBox"
```

Comment:
In general, this groupbox and its contents should be padded/margin'd the same way as the groupbox below ("Select public members to form interface"). So, for example, the contents (radio buttons) are too close to the groupbox border on all sides, and the radio buttons are also probably too close to each other since currently the text box basically touches the 'g' in "existing file". Also, the right side of the "New file name" text box should align with the right side of the "Select All" button if padded correctly.